### DISCUSSION TITLE
Whats the difference between the Ensemble Algorithms?

### DISCUSSION THREAD

**Discussion by xzuyn** [Anjok07/ultimatevocalremovergui#462] [Q&A]
Are there any explanations on what these different options do?

![ensemble](https://user-images.githubusercontent.com/16216325/227498530-a242b6e5-eccf-4da5-8d52-daf8cd8e28a6.png)

**Comment by LucidFir** [comment]
https://www.youtube.com/watch?v=ykOKwz3eRUQ&ab_channel=soundlearn

**Reply by zzryn** [reply]
Thanks! You forgot a timestamp, so the discussion for ensemble algos are at 4:30.
